### ISSUE TITLE
Request - "synthesize_csv.py" from YouTube "Coqui TTS Audio samples of all models (Version 0.7.1)"

### ISSUE THREAD

**Issue by rekab72** [thorstenMueller/Thorsten-Voice#51]
There are now 66 models now available to Coqui TTS, 18 are English. I would like to run your python script with the new models. Can you post your script and a sample csv to modify?

**Comment by thorstenMueller** [comment]
Okay, i'll search for it and publish it (if i can find it 😆).
Just to be sure, you mean my script magic from here? https://youtu.be/HojuVmW5LUI?t=486

**Comment by rekab72** [comment]
Yes, please! It is of interest to me to hear all the samples of -
`--model_name tts_models/en/vctk/vits --speaker_idx <speaker id>`. 
It seems there are quite a few <speaker id> p225 - p376, all with varying speakers, tones, speeds.
I haven't found an easy reference of all that have been created EXCEPT, you of course, who posted samples of the models available at the time of that video. Thank you!

**Comment by thorstenMueller** [comment]
Oops @rekab72 , somehow the request was forgotten. Is there still interest in the script?

**Comment by rekab72** [comment]
Yes, I have time now to review. Thanks

-- 
 Sent with Tuta; enjoy secure & ad-free emails: 
 https://tuta.com


Mar 16, 2024, 03:45 by ***@***.***:
